lab/homework/test.js

```diff
@@ -0,0 +1,97 @@
+import {expect} from 'chai';
+import {HomeworkError, Product, Aliment, Furniture, Person, Customer, Cashier} from './index.js';
+
+
+
+
+
+describe('Customer ERRORS', () => {
+
+    it('a customer won\'t add to his shopping cart an aliment that is expired', () => {
+        let customer1 = new Customer({money: 100, shoppingCart: new Array()});
+        let prod0 = new Furniture({name: "sofa", price: 70, colour: "blue"});
+        let prod1 = new Aliment({name: "pudding", price: 10, expirationDate: new Date() + 1, grammage: 50});
+        let prod2 = new Aliment({name: "puddingx3", price: 30, expirationDate: new Date() + 1, grammage: 150});
+        let prod3 = new Aliment({name: "puddingx3", price: 30, expirationDate: new Date() - 7, grammage: 150});
```

Comment:
Take care that:

1. `new Date() - 7` converts the Date instance to number, namely how many milliseconds passed from 01.01.1970 until present (UNIX timestamps) and then performs arithmetic.

2.  `new Date() + 1` converts the Date instance to its string representation and then concatenates it with the other operand.

You should use `Date.now()` if you want to operate on UNIX timestamps and make sure arithmetic works fine.